Update 15 (#277)
NilPointer-Software/bdfd-wiki: src/bdscript/channelPositionComplex.md

# $channelPosition[]

> Returns the position of a specified channel in the server using it's ID.

Reviewer: ```suggestion
Returns the position of a given channel using its ID.

```

## Syntax
```
$channelPosition[channel ID]
```

> ### Parameters
> - `channel ID` `(Type: Snowflake || Flag: Required)`: The ID of a specific channel to get it's position.

Reviewer: ```suggestion

### Parameters
- `channel ID` `(Type: Snowflake || Flag: Required)`: The ID of a channel whose position will be returned.
```
